### ISSUE TITLE
Cross-post the xvec blog post from Earthmover blog?

### ISSUE THREAD

**Issue by TomNicholas** [xarray-contrib/xarray.dev#693]
I just saw the ["Vector data cubes in xarray"](https://earthmover.io/blog/vector-datacube-pt1) blog post - awesome content!

Any chance this can be cross-posted to the xarray blog here too as a guest post? AFAIK most of the xvec development work was done independently of Earthmover, and this is certainly of interest to xarray users who don't use arraylake.

cc @dcherian @e-marshall

**Comment by dcherian** [comment]
I had suggested just making the last bit an Xvec example notebook.

**Comment by TomNicholas** [comment]
Sure, but this is effectively an xarray integration feature announcement, and we have a pattern of posting content like that to the xarray blog (e.g. the pint-xarray stuff) - it would be nice to continue doing that too.

**Comment by dcherian** [comment]
Oh I did not realize that we haven't had an xvec blogpost yet!

@martinfleis @benbovy  : would you be OK with the cross-post here or would you prefer to add a separate post introducing Xvec.

**Comment by TomNicholas** [comment]
> Oh I did not realize that we haven't had an xvec blogpost yet!

I don't think we have really had anything that shows off the flexible indexes stuff yet! (Only a mention of work in progess on @keewis ' work on the pint index in the pint-xarray blog post almost 2 years ago!).

**Comment by keewis** [comment]
we have a draft here: #597. We now have at least three examples of flexible indexes, so maybe it's time to finish that?

For reference, the ones that I'm aware of are the `xvec` index, `PintIndex`, and `DGGSIndex`.

**Comment by TomNicholas** [comment]
> so maybe it's time to finish that?

I think so, but I don't think we should wait to do that before cross-posting the xvec blog post on the xarray blog.

**Comment by e-marshall** [comment]
I'd be very happy to cross-post on the Xarray blog if folks are interested in that!

**Comment by martinfleis** [comment]
I have "write xvec blog post" on my to-do for over a year and it hasn't happened yet. It eventually will but it may be wiser to just use this one for xarray blog.

**Comment by benbovy** [comment]
Good idea and agreed we shouldn't wait for a post on flexible indexes.

#597 has stalled (like other in-progress work on flexible indexes) but it will be resumed. I'll probably split it into several smaller posts.

**Comment by dcherian** [comment]
In that case I propose:
1.  @e-marshall copies over the post, 
2. we add @martinfleis and @benbovy as co-authors, 
3. @martinfleis and @benbovy make edits/changes.
4. we merge and publish.

how does that sound?

**Comment by martinfleis** [comment]
I can try to expand it to cover a bit more about Xvec.

**Comment by e-marshall** [comment]
I copied the blog post [here](https://github.com/xarray-contrib/xarray.dev/pull/701), feel free to edit as needed and i'm happy to work on formatting/ wording changes as well

**Comment by martinfleis** [comment]
A bit more stuff added on top of @e-marshall's text in https://github.com/e-marshall/xarray.dev/pull/1.
